### ISSUE TITLE
Missing cbor dependency on Windows

### ISSUE THREAD

**Issue by robin-nitrokey** [Nitrokey/pynitrokey#164]
[Original report](https://support.nitrokey.com/t/nitropy-lasst-sich-auf-windows-10-nicht-starten/3815)

```
D:\tmp>nitropy-0.4.13.exe
Traceback (most recent call last):
  File "nitropy.py", line 13, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 359, in exec_module
  File "pynitrokey\cli\__init__.py", line 18, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 359, in exec_module
  File "pynitrokey\cli\fido2.py", line 19, in <module>
ModuleNotFoundError: No module named 'cbor'
[23860] Failed to execute script nitropy
```

Observations:
- `cbor` is present in the `pyproject.toml` dependency list but not in `ci-requirements.txt`.
- [`wine-build/build-wine-docker.sh`](https://github.com/Nitrokey/pynitrokey/blob/master/wine-build/build-wine-docker.sh#L103-L108) should use both, but maybe the dependency installation with `flit` does not work properly.

**Comment by robin-nitrokey** [comment]
From the build logs:
```
+ py -m flit install
+ wine c:/python3.6.8/python.exe -O -B -m flit install
0075:fixme:msvcrt:_configure_wide_argv (1) stub
0075:fixme:msvcrt:_initialize_wide_environment stub
c:\python3.6.8\python.exe: No module named flit
```

**Comment by robin-nitrokey** [comment]
Fixed in #165.
